packages/angular/cli/models/schematic-command.ts

```diff
@@ -460,7 +460,8 @@ export abstract class SchematicCommand<
     let input = { ...pathOptions, ...args };
 
     // Read the default values from the workspace.
-    const projectName = input.project !== undefined ? '' + input.project : null;
+    const projectName = input.project !== undefined
```

Comment:
I don’t think this is a fair assumption. Probably the current project should taken based on the cwd. That’s said for `ng build` at the moment it uses the defaultProject when no project is supplied.

This I think opens up a broader discussion.

//cc @clydin.

Reply:
If the command was called from the workspace root path, the variable ```projectName``` will be null (this is what is happen right now).
This behavior creates a bug since the default project configurations are completely ignored.